docs/src/staking/stake-accounts.md

```diff
@@ -0,0 +1,90 @@
+the wallet, the keypair associated with a stake account address does not have
+any control over the account.  In fact, a keypair or private key may not even
+exist for a stake account's address.
+
+The only time a stake account's address has a keypair file is when [creating
+a stake account using the command line tools](../cli/delegate-stake.md#create-a-stake-account),
+a new keypair file is created first only to ensure that the stake account's
+address is new and unique.
+After the account is created at its address, the keypair file that was used to
+derive the address can be discarded.
+
+#### Understanding Account Authorities
+Each stake account has two signing authorities specified by their respective address,
+each of which is authorized to perform certain operations on the stake account.
+
+The *stake authority* is used to sign transactions for the following operations:
+ - Delegating stake
+ - Deactivating the stake delegation
+ - Splitting the stake account, creating a new stake account with a portion of the
+ funds in the first account
+ - Setting a new stake authority
+
+The *withdraw authority* signs transactions for the following:
+ - Withdrawing un-delegated stake into a wallet address
+ - Setting a new withdraw authority
+ - Setting a new stake authority
+
+The stake authority and withdraw authority are set when the stake account is
+created, and they can be changed to authorize a new signing address at any time.
+The stake and withdraw authority can be the same address or two different
+addresses.
+
+The withdraw authority keypair holds more control over the account as it is
+needed to liquidate the tokens in the stake account, and can be used to reset
+the stake authority if the stake authority keypair becomes lost or compromised.
+
+#### Multiple Delegations
+A stake account may only ever be delegated to one validator at a time. All of
+the tokens in the account are either delegated or un-delegated, or in the
+process of becoming delegated or un-delegated.  To delegate a fraction of your
+tokens to a validator, or to delegate to multiple validators, you must create
+multiple stake accounts.
+
+This can be accomplished by creating multiple stake accounts from a wallet
+address containing some tokens, or by creating a single large stake account
+and using the stake authority to split the account into multiple accounts
+with token balances of your choosing.
+
+Stake and withdraw authority addresses can be assigned to multiple stake accounts.
+
+#### Delegation Warmup and Cooldown
+When a stake account is delegated, or a delegation is deactivated, the operation
+does not take effect immediately.
+
+A delegation or deactivation takes several epochs to complete, with a fraction
+of the delegation becoming active or inactive at each epoch boundary after
+the transaction containing the instructions has been submitted to the cluster.
+
+There is also a limit on how much global stake can become delegated or
```

Comment:
nits: `global` -> `cluster-wide`? I know this doc doesn't require strongest rigidness.. so `nits:` :)